src/Land/Model/prog_types.jl

```diff
@@ -0,0 +1,7 @@
+#####
+##### Prognostic Variable types
+#####
+
+struct ϑLiquid <: AbstractPrognosticVariable end
+struct InternalEnergy <: AbstractPrognosticVariable end
```

Comment:
Should we be careful that this is a volumetric internal energy, as opposed to a specific internal energy?

Reply:
This is basically a mirror from
```julia
vars_state(heat::SoilHeatModel, st::Prognostic, FT) = @vars(ρe_int::FT)
```
and
```julia
vars_state(water::SoilWaterModel, st::Prognostic, FT) = @vars(ϑ_l::FT, θ_i::FT)
```
I'm not sure if there are better names out there, but that's the basic idea.

Reply:
I was thinking back to the discussion we had when we first added this variable. variables with name `e_` denote a specific energy per unit volume, `E_` denotes energy, and here we have `rhoe_`, denoting energy per unit volume. Since we took the care there to be precise, should this be named something like `VolumetricInternalEnergy` or `InternalEnergyDensity` instead?

Reply:
Either one seems to me